### ISSUE TITLE
NVDA automatically cancel reporting of current application title after selecting character from chinese input 

### ISSUE THREAD

**Issue by LittleStar-VIP** [nvaccess/nvda#16242]
### Is your feature request related to a problem? Please describe.
*this issue is also recently discussed in the nvda-tw mailing list
when I try to use the chinese input '微軟倉頡輸入法' to input a chinese character, the input method may provide character suggestion for me to choose from.
In windows 11, whenever I have selected an option from the character suggestion popup, nvda will always report the title again before reporting what I have selected.
Base on my initial test, this is not a problem limited to nvda, and narrator also has this problem.
below I illustrate the issue using an example:
1. open notepadd++
2. use the above mentioned chinese input method to type大月and type space
3. a character suggestion popup, saying '候選字 清單 1有2冇'
4. then i press 1 to select 有
5. nvda reports '*new 1 - Notepad++ ... 有'

actually this problem appears in every applications which has text input box.
since it repeat the window title everytime I have selected the character, which wasted me lots of time waiting for nvda to report what character has been selected.

Below are the basic info of my system:
Windows 11 Home 22H2
NVDA 2023.3.3 installed version, and have tested with nvda 2024.1 beta 10

### Describe the solution you'd like
according the process name detected by nvda profile window, the character suggestion popup has the process name of 'textinputhost'
can nvda automatically detect if last focused window is 'textinputhost, then nvda will automatically canceled the reporting of title?

**Comment by LittleStar-VIP** [comment]
this suggestion is related to another discussion #14838

**Comment by Adriani90** [comment]
cc: @josephsl

**Comment by josephsl** [comment]
Hi,

Cause: focus events. When IME window closes, it looks like a foreground event is recorded by NVDA (tested with Event Tracker ad-on with Windows App Essentials disabled). As this affects Narrator as well, this might be due to modern keyboard interface design. Unless Microsoft notices issues with how Narrator handles this one, I doubt this can be fied anytime soon (this is not relaly an NVDA issue also, I think).

Thanks.

**Comment by LittleStar-VIP** [comment]
@josephsl 
Yes I totally understood this is not a nvda issue, as I have already stated in the original post, I am just proposing this see if nvda can provide a work around.

And not sure if you know, for this kind of issue, where can i submit the bug report to microsoft?

**Comment by Adriani90** [comment]
I made good experiences with the Microsoft Disability Answer Desk.If you google it and open that website, there is a heading called „ chat with us“ after which you find the link to launch the live chat with accessibility experts.https://www.microsoft.com/en-us/accessibility/disability-answer-deskVon meinem iPhone gesendetAm 04.03.2024 um 00:16 schrieb LITTLE STAR☆VIP ***@***.***>:﻿
@josephsl
Yes I totally understood this is not a nvda issue, as I have already stated in the original post, I am just proposing this see if nvda can provide a work around.
And not sure if you know, for this kind of issue, where can i submit the bug report to microsoft?

**Comment by Adriani90** [comment]
@josephsl is there any improvement in this regard which came after #16541? This is now merged into master.

**Comment by josephsl** [comment]
Hi, don’t think so as I addressed IME candidate announcements in Windows 11, not this one (unless it is a side effect). Let’s wait for answers from Chinese speakers. Thanks.

**Comment by LittleStar-VIP** [comment]
@Adriani90 
based on my initial test of the pr build in #16541
the issue recorded in this post has no improvement.
